src/lib/user-ip-address.js

```diff
@@ -0,0 +1,16 @@
+module.exports = (req) => {
+  const xHeaderIpAddress = req.headers["x-forwarded-for"];
+
+  let ipAddress;
+  if (xHeaderIpAddress !== undefined) {
+    ipAddress = xHeaderIpAddress.split(",")[0];
+  } else {
```

Comment:
@praveen-gds is there a test case checking the behaviour in this else block?

Reply:
yes added it.